docs/examples/custom_destination_lancedb/.dlt/example.secrets.toml

```diff
@@ -0,0 +1,3 @@
+[spotify]
```

Comment:
whose spotify secrets will we be using on CI?

Reply:
We can use mine. WIll message on slack